tensorflow/python/kernel_tests/slice_op_test.py

```diff
@@ -28,8 +28,9 @@
 from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import gradients_impl
 from tensorflow.python.ops import math_ops
+from tensorflow.python.ops import nn_ops
 from tensorflow.python.platform import test
-
+from tensorflow.python.ops import random_ops
```

Comment:
This line should be above `from tensorflow.python.platform import test`

Reply:
@penpornk OK, fixed.